packages/opentelemetry/src/utils/parseSpanDescription.ts

```diff
@@ -43,9 +44,14 @@ export function parseSpanDescription(span: AbstractSpan): SpanDescription {
     return descriptionForHttpMethod({ attributes, name, kind: getSpanKind(span) }, httpMethod);
   }
 
-  // If db.type exists then this is a database call span.
   const dbSystem = attributes[SEMATTRS_DB_SYSTEM];
-  if (dbSystem) {
+  const opIsCache =
+    typeof attributes[SEMANTIC_ATTRIBUTE_SENTRY_OP] === 'string' &&
```

Comment:
did we end up reverting to this on purpose, vs. doing `opIsCache = typeof attributes['cache.hit'] === 'boolean'`? :) If we can avoid string operations here I think it would be preferable!

Reply:
Yes I reverted it as `cache.hit` is only added for a `get` operation.